types/file_name.pp

```diff
@@ -0,0 +1,2 @@
+# The name of a file. Not a full path!
```

Comment:
please use `@summary` here as well so puppet-strings can pick it up

Reply:
fixed, REFERENCE.md updated